README.md

```diff
@@ -79,9 +79,33 @@ In graph theory terminology, they are `leaf` nodes of your graph.
 
 Run `utensor-cli convert --help` for detailed information.
 
-# Example
+# Use `utensor_cgen` as Library
+
+## Subgraph Isomorphic Matcher and Nodes Fusion
+
+With `uTensorGraphMatcher`, performing a subgraph isomorphic match, replace or manipulate the matched subgraph is just a few line of code!
```

Comment:
With `uTensorGraphMatcher`, performing common subgraph tasks such as isomorphic matching along with replacing or manipulating the matched subgraph(s) takes just a few line of code!

Reply:
Better.

Reply:
We should make a note here that future transformers will be based on our own graph-constructors/builders.

Reply:
We should make a note here that future transforms will be based on our own graph constructor/builder.